frontend/src/bundles/common/components/radio/radio-group.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+    FormControlLabel,
+    RadioGroup as MuiRadioGroup,
+    type SxProps,
+    type Theme,
+} from '@mui/material';
+
+import { type Color } from '~/bundles/common/enums/enums.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { Radio } from './radio-item.js';
+
+type Option = {
```

Comment:
I this we can export. Because we will use it, when we will use this component

Reply:
@nikita-remeslov But I exported _both Radio and RadioGroup_ in **radio.tsx** Or it is better to do it here ?

Reply:
you are right!